src/main/java/frc/robot/constants/LocalConstants.java

```diff
@@ -92,7 +92,7 @@ public LocalSwerveConstants() {
             public LocalSwerveModuleConstants rearLeftModuleConstants;
 
             public static class LocalSwerveModuleConstants {
-                double encoderOffset;
+                public double encoderOffset;
```

Comment:
docs?

Reply:
maybe they all should be public or something